tests/integration/configs/configs.spec.ts

```diff
@@ -156,6 +158,199 @@ describe('config', function () {
+        version: 1,
+        config: {
+          manager: {
+            $ref: {
+              configName: 'latest-ref-target',
+              version: 'latest',
+              // Note: missing schemaId - this is v1 format
+            },
+          },
+          role: 'manager',
+        },
+        createdBy: 'system',
+        isLatest: true,
+        configSchemaVersion: 'v1',
+      };
+
+      await drizzle.insert(configs).values(configWithLatestRef);
+
+      await configManager.updateOldConfigs();
+
+      // Verify the config was updated with v2 format refs including schemaId
+      const updatedConfig = await configManager.getConfig('config-with-latest-ref', 'https://mapcolonies.com/schemaWithRef/v1');
+      expect(updatedConfig).toHaveProperty('configSchemaVersion', 'v2');
+
+      const configData = updatedConfig.config;
+      const manager = configData.manager as Record<string, unknown>;
+      const managerRef = manager.$ref as Record<string, unknown>;
+      expect(managerRef).toHaveProperty('configName', 'latest-ref-target');
+      expect(managerRef).toHaveProperty('version', 'latest');
+      expect(managerRef).toHaveProperty('schemaId', 'https://mapcolonies.com/simpleSchema/v1');
+    });
+
+    it('should continue processing other configs when one config update fails', async function () {
+      const drizzle = dependencyContainer.resolve<Drizzle>(SERVICES.DRIZZLE);
+      const configManager = dependencyContainer.resolve(ConfigManager);
+      const configRepository = dependencyContainer.resolve(ConfigRepository);
+
+      // Generated by Copilot
+      // Insert multiple configs with v1 schema version
+      const oldConfigs = [
+        {
+          configName: 'config-will-fail',
+          schemaId: 'https://mapcolonies.com/simpleSchema/v1',
+          version: 1,
+          config: { name: 'failing', age: 40 },
+          createdBy: 'system',
+          isLatest: true,
+          configSchemaVersion: 'v1',
+        },
+        {
+          configName: 'config-will-succeed',
+          schemaId: 'https://mapcolonies.com/simpleSchema/v1',
+          version: 1,
+          config: { name: 'succeeding', age: 45 },
+          createdBy: 'system',
+          isLatest: true,
+          configSchemaVersion: 'v1',
+        },
+      ];
+
+      await drizzle.insert(configs).values(oldConfigs);
+
+      // Mock updateConfigToNewSchemaVersion to fail for first config
+      let callCount = 0;
+      jest.spyOn(configRepository, 'updateConfigToNewSchemaVersion').mockImplementation(async (input) => {
+        callCount++;
+        if (callCount === 1 && input.configName === 'config-will-fail') {
+          throw new Error('Simulated database error');
+        }
+        // For subsequent calls, just resolve successfully since we're testing error handling
+        return Promise.resolve();
+      });
+
+      // Should not throw even if one config fails
+      await expect(configManager.updateOldConfigs()).resolves.not.toThrow();
```

Comment:
if you already has jest-extended, it will be more nice to use ".toResolve()"
https://www.npmjs.com/package/jest-extended/v/2.0.0#toresolve